hc/api/tests/test_list_checks.py

```diff
@@ -33,15 +33,30 @@ def get(self):
 
     def test_it_works(self):
         r = self.get()
-        ### Assert the response status code
+        self.assertEqual(r.status_code, 200)
 
         doc = r.json()
         self.assertTrue("checks" in doc)
 
         checks = {check["name"]: check for check in doc["checks"]}
-        ### Assert the expected length of checks
-        ### Assert the checks Alice 1 and Alice 2's timeout, grace, ping_url, status,
-        ### last_ping, n_pings and pause_url
+        self.assertEqual(len(checks),2)
+        self.assertIn('Alice 1',checks.keys() )
+        self.assertEqual(checks[self.a1.name]['timeout'],3600)
+        self.assertEqual(checks[self.a1.name]['grace'],900)
+        self.assertNotEqual(checks[self.a1.name]['pause_url'],0)
+        self.assertNotEqual(len(checks[self.a1.name]['ping_url']),0)
```

Comment:
Would be better to check the 2 urls are not `None`. 
Or append a `/pause` to the `ping_url` and see if it'll give you the `pause_url`